### PR TITLE
Intro css styling

### PR DESCRIPTION
**Pull request by JM066**

### CODE REVIEW

**Review by ivanms1 — CHANGES_REQUESTED** [review]
also check conflicts

**Review by BruceJi7 — APPROVED** [review]

File: src/components/Form/HookForm/FormInput/index.tsx [hunk 1]
```diff
@@ -4,15 +4,15 @@ import { FieldError } from "react-hook-form";
 
 import styles from "./FormInput.module.scss";
 
-import inputStyles from "../../Input/Input.module.scss";
+// import inputStyles from "../../Input/Input.module.scss";
```
Comment (ivanms1) [inline src/components/Form/HookForm/FormInput/index.tsx:7]: remove please

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -1,55 +1,59 @@
...
 
-function Header() {
-  const [network, setNetwork] = useState("testnet");
-  const currentStep = useStore((state) => state.currentStep);
+const BLUE_SELECT_STEPS = ["helpUs"];
 
+function Network() {
+  const currentStep = useStore((store) => store.currentStep);
+  const currentNetwork = useStore((store) => store.currentNetwork);
   const isSelectBlue = BLUE_SELECT_STEPS.includes(currentStep.current);
```
Comment (ivanms1) [inline src/components/Header/index.tsx:18]: add a space to separate from hooks

File: src/components/Header/index.tsx [hunk 3]
```diff
@@ -1,55 +1,59 @@
...
+  );
+}
+
+function Header() {
+  const currentStep = useStore((store) => store.currentStep);
+  const isUserLoggedIn = useStore((state) => state.isUserLoggedIn);
+
+  if (!isUserLoggedIn && currentStep.current === "welcome") {
+    return null;
+  }
```
Comment (ivanms1) [inline src/components/Header/index.tsx:40]: add space

File: src/pages/Intro/Backup/ModalConfirmation/ModalConfirmation.module.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
+.ConfirmModal {
+  border-radius: 10px;
+  top: 30%;
+  width: 320px;
+  & .ConfirmContainer {
```
Comment (ivanms1) [inline src/pages/Intro/Backup/ModalConfirmation/ModalConfirmation.module.scss:5]: add space between class names

File: src/pages/Intro/HelpUs/index.tsx [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
 import useStore from "@/store/store";
 
 import GoBack from "@/assets/icons/go-back.svg";
 import GreenCircle from "@/assets/icons/green_circle.svg";
 import RedCircle from "@/assets/icons/red_circle.svg";
+
 import styles from "./HelpUs.module.scss";
 
 function HelpUs() {
   const setIsLoggerActive = useStore((state) => state.setIsLoggerActive);
```
Comment (ivanms1) [inline src/pages/Intro/HelpUs/index.tsx:13]:
```suggestion
  const setIsLoggerActive = useStore((store) => store.setIsLoggerActive);
```

File: src/pages/Intro/ImportWallet/index.tsx [hunk 6]
```diff
@@ -31,7 +32,10 @@ function ImportWallet() {
       toast.error("Please upload an id file");
       return;
     }
-
+    if (!isChecked) {
+      toast.error("Please check to agree to our Terms and conditions");
+      return;
```
Comment (ivanms1) [inline src/pages/Intro/ImportWallet/index.tsx:37]: use `else`

File: src/pages/Intro/WalletQuestion/index.tsx [hunk 7]
```diff
@@ -9,16 +9,16 @@ import styles from "./WalletQuestion.module.scss";
 
 function WalletQuestion() {
   const setCreateNewWallet = useStore((state) => state.setCreateNewWallet);
-  const setCurrentStep = useStore((state) => state.setCurrentStep);
-
+  const setCurrentStep = useStore((store) => store.setCurrentStep);
+  const currentStep = useStore((state) => state.currentStep);
```
Comment (ivanms1) [inline src/pages/Intro/WalletQuestion/index.tsx:13]:
```suggestion
  const currentStep = useStore((store) => store.currentStep);
```

**Review by ivanms1 — CHANGES_REQUESTED** [review]

**Review by ivanms1 — APPROVED** [review]

**Comment by ivanms1** [comment]
merge once conflicts are solved @JM066

**Pull request merged**